source/_integrations/reolink.markdown

```diff
@@ -204,6 +204,18 @@ However, these cameras can work with this integration through an NVR in which th
 - E1 Pro
 - E1
 
+## Initial Setup
+
```

Comment:
I would first recommend users to just use the Reolink app/windows/mac client and follow the instructions there. That is by far the easiest and what most users are going to do. Once the setup is complete and the username and password of the camera is set in the Reolink client, use those same credentials inside Home Assistant.

Then as a second option for camera's that have a LAN port (most WiFi camera's also have a LAN port), connect the camera to your network using a LAN cable and browse to the IP adress of the camera. Follow the instructions to create the user credentials (needed in HomeAssistant). Then in the settings of the camera, configure the wifi SSID and password, once configured disconnect the LAN cable and the camera will automatically switch to the wifi connection if the wifi credentials are correct.

Then we can explain how to do it if someone insists on not using the Reolink app/windows/mac client and has a camera withouth a LAN port as you discribed.